src/relay.js

```diff
@@ -288,12 +292,12 @@ export function sequelizeConnection({
         // In case of `OVER()` is not available, we need to get the full count from a second query.
         const options = await Promise.resolve(before({
           where: argsToWhere(args)
-        }));
+        }, args, context, info));
 
         if (target.count) {
-          fullCount = await target.count(source, options);
+          fullCount = await target.count(options);
         } else {
-          fullCount = await target.manyFromSource.count(source, options);
+          fullCount = await target.manyFromSource.count(options);
```

Comment:
@mickhansen Sure. I'm talking about fixing this while providing support for `target: Model`, which is my case. Do you want me to fix that ?